codegen/src/main/twirl/templates/PlayScalaServer/Router.scala.txt

```diff
@@ -0,0 +1,37 @@
+@*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ *@
+
+@(service: akka.grpc.gen.scaladsl.Service)
+
+package @service.packageName
+
+import javax.inject._
+
+import scala.concurrent.Future
+
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+import akka.stream.Materializer
+
+import play.api.mvc._
+import play.api.mvc.akkahttp.AkkaHttpHandler
+import play.api.routing.Router
+import play.api.routing.Router.Routes
+
+@@Singleton
+class @{service.name}Router @@Inject()(cc: ControllerComponents, impl: controllers.@{service.name}Impl)(implicit mat: Materializer)
```

Comment:
We don't need to inject `ControllerComponents` here since we're not doing building an action, etc.